hazelcast/src/hazelcast/client/internal/socket/TcpSocket.cpp

```diff
@@ -296,6 +283,51 @@ namespace hazelcast {
                         return std::auto_ptr<Address>();
                     }
                 }
+
+                SocketInterface &TcpSocket::setSocketOptions(const client::config::SocketOptions &socketOptions) {
+                    int optionValue = socketOptions.getBufferSize();
```

Comment:
I think for better readability instead of using `optionValue` we can use directly the option's name, for example here `int bufferSizeInKB = socketOptions.getBufferSizeInKB()`

Reply:
I named it this way to reuse the same local variable for each option instead of stack allocating a new one for each.